MapboxCoreNavigation/DistanceFormatter.swift

```diff
@@ -6,32 +7,32 @@ let yardsPerMile = 1_760.0
 let feetPerMile = yardsPerMile * 3.0
 
 /// Provides appropriately formatted, localized descriptions of linear distances.
-class DistanceFormatter: LengthFormatter {
+public class DistanceFormatter: LengthFormatter {
     /// True to favor brevity over precision.
-    var approximate: Bool
+    var approx: Bool
```

Comment:
What does this change have to do with renaming the repositories? Can it be split into a separate PR?

(Also, I like the irony of abbreviating a property described as “favoring brevity over precision”. 😄)